src/platform/silabs/efr32/efr32-chip-mbedtls-config.h

```diff
@@ -42,6 +42,34 @@
 #define MBEDTLS_SSL_OUT_CONTENT_LEN 1560
 #define MBEDTLS_PSA_BUILTIN_ALG_TLS12_PRF
 #define MBEDTLS_MPI_MAX_SIZE 512
+#define MBEDTLS_CTR_DRBG_C
```

Comment:
Do you need al those for your DIC use case? We purged some we didn't need to reduce the code size. Also most configs. now comes from the slc generate board configs see an example here
```
third_party/silabs/matter_support/matter/efr32/efr32mg24/BRD4186C/config/psa_crypto_config.h
third_party/silabs/matter_support/matter/efr32/efr32mg24/BRD4186C/config/sl_mbedtls_config.h
```

Reply:
@jmartinez-silabs  We removed some macros which is not required for DIC from this list.

Reply:
I agree with Junior on that one, please encase all of this under the `DIC` define (#ifdef DIC) so that the code size won't be bloated for all sample apps.

Reply:
Also since the source files is included conditionally to the DIC feature, I'm surprised that we are not hitting any issue with non DIC applications 🤔

Reply:
@jepenven-silabs we added all these macros under DIC macro only

Reply:
didn't see it at first 😮